code/game/gamemodes/meteor/meteor_object.dm

```diff
@@ -84,14 +84,14 @@ GLOBAL_LIST_EMPTY(asteroids)
 	//That includes empty space with potential non empty space below
 	var/impacting = FALSE
 	if (istype(next_loc))
-		if (istype(next_loc, /turf/space))
+		if (next_loc.z_eventually_space)
```

Comment:
this doesn't seem right. 

if z_eventually_space is true, then we shouldn't need to do anymore checks, we can assume the turf is fine to move into ?
it should be true in cases where the turf is space tiles all the way down